SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/VisibilityTest/VisibilityTest.h

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "CoreMinimal.h"
+#include "SpatialFunctionalTest.h"
+#include "VisibilityTest.generated.h"
+
+class ATestMovementCharacter;
+
+UCLASS()
+class SPATIALGDKFUNCTIONALTESTS_API AVisibilityTest : public ASpatialFunctionalTest
+{
+	GENERATED_BODY()
+public:
+	AVisibilityTest();
+
+	virtual void BeginPlay() override;
+	void GetLifetimeReplicatedProps(TArray<FLifetimeProperty>& OutLifetimeProps) const override;
+
+	UPROPERTY(Replicated)
+	TArray<APlayerController*> Controllers;
```

Comment:
It seems like you are not using this in the test itself, so might want to remove it?
```suggestion
```